setup.py

```diff
@@ -45,8 +45,8 @@
         'gym>=0.17.2',
         'mujoco-py>=2.0.2.10',
         'numpy>=1.17.5',
-        'pandas',
-        'ray[tune]>=0.8.0',
+        'pandas>=1.0.3',
```

Comment:
The pandas features we use are so trivial (don't actually know if any of the current code uses them), that any older version of pandas should work. It's better to leave this without version since otherwise this will possibly override all the other libraries' pandas version, which we don't want to do.
```suggestion
        'pandas',
```

Reply:
Okay, it sounds alright to make this revision. The requirements.txt references pandas==1.0.3, so I thought for consistency we might want to include that version in the setup.py. But if pandas is not being used for any version-specific features, this might not be an important change to make.

https://github.com/rail-berkeley/softlearning/blob/master/requirements.txt#L62